### PR TITLE
improvements

### PR DESCRIPTION
**Pull request by frangametv**

### CODE REVIEW

**Comment by pH-7** [comment]
Thanks very much for your contribution @frangametv 

Can you describe what your PR does and add some screenshots of the text editor after your changes?

**Comment by frangametv** [comment]
I moved the "icons" folder inside the .jar and used class resource loading; then i forced the JavaAWT UIManager Style to Windows Modern Look, and this gives a completely new modern look to the text editor.
You have the code, I only changed a few lines and I also recompiled the project into a .jar, you can see the differences by yourself.

File: src/simplejavatexteditor/UI.java [hunk 1]
```diff
@@ -1,40 +1,3 @@
-/**
- * @name        Simple Java NotePad
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Could you put back this comment please?

File: .classpath [hunk 2]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="src" path="icons"/>
```
Comment (pH-7) [inline .classpath:4]: Was this redundant?

**Review by pH-7 — CHANGES_REQUESTED** [review]
👍 Just two minor changes needed to be confirmed/changed.

**Comment by pH-7** [comment]
@frangametv With what Java version did you test it?

@frangametv Could you also update the `SimpleJavaTextEditor.jar` with your version and run it? 
You can also generate easily a new jar file with the following command when you are in src/ directory `jar cmvf ../manifest.mf ../SimpleJavaTextEditor.jar simplejavatexteditor/*.class`

File: .classpath (hunk 2 above)
Comment (frangametv) [reply .classpath:4]: Yes

**Comment by frangametv** [comment]
**Java version:** From 1.8 to the newest, it worked just fine
I already updated the Jar file, as you can see from the changes.

**Comment by frangametv** [comment]
Sorry for the deleted comment, my bad

**Comment by pH-7** [comment]
> **Java version:** From 1.8 to the newest, it worked just fine
> I already updated the Jar file, as you can see from the changes.

Beautiful! This sounds brilliant @frangametv! I will check this out this evening.

File: .classpath (hunk 2 above)
Comment (pH-7) [reply .classpath:4]: 👍

File: src/simplejavatexteditor/SimpleJavaTextEditor.java [hunk 3]
```diff
@@ -32,6 +34,13 @@ public class SimpleJavaTextEditor extends JTextPane {
      * @param args
      */
     public static void main(String[] args) {
+        //UIMANAGER
```
Comment (pH-7) [inline src/simplejavatexteditor/SimpleJavaTextEditor.java]: I think this comment is quite redundant 🙃  as we can see in the try block we use UIManager (unless you had something else in mind you wanted to let know in the comment..)

File: src/simplejavatexteditor/UI.java [hunk 4]
```diff
@@ -73,39 +73,39 @@ public class UI extends JFrame implements ActionListener {
...
+    private final ImageIcon aboutIcon = new ImageIcon(UI.class.getResource("icons/about.png"));
 
     private SupportedKeywords kw = new SupportedKeywords();
     private HighlightText languageHighlighter = new HighlightText(Color.GRAY);
     AutoComplete autocomplete;
     private boolean hasListener = false;
     private boolean edit = false;
 
-    public UI() {
+    public UI() {
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: No biggie, just extra spaces here :)

**Comment by frangametv** [comment]
fixed

**Comment by pH-7** [comment]
> fixed

This is brilliant! Amazing job!

**Review by pH-7 — APPROVED** [review]

**Comment by pH-7** [comment]
I’ve now merged your changes. Thanks again for your contribution to this project. I really appreciate the improvements you have done to it 👍

**Pull request merged**